tests/adapters/compliance_tests/unset_interface_state_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2016 Internap.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from hamcrest import assert_that, is_
+from netman.core.objects.exceptions import UnknownInterface
+from netman.core.objects.interface_states import ON, OFF
+
+from tests.adapters.configured_test_case import ConfiguredTestCase
+
+
+class UnsetInterfaceStateTest(ConfiguredTestCase):
+    _dev_sample = "juniper"
+
+    def test_returns_an_interface_to_default_state(self):
+
+        default_state = self.client.get_interface(self.test_port).shutdown
```

Comment:
Shouldn't that member be called "state" if the action to change it is "{un,}set_interface_state"?

Not blocking the PR for this.